src/main.py

```diff
@@ -38,7 +38,19 @@ def build_change_set(config, disable_ssl_verification):
         base_url=os.environ.get("DBT_BASE_URL", "https://cloud.getdbt.com"),
         disable_ssl_verification=disable_ssl_verification,
     )
-    cloud_jobs = dbt_cloud.get_jobs()
+
+    # If a project_id or environment_id is passed in as a parameter, check if it is the same in Jobs YAML file, otherwise raise an error
+    if (project_id != 0 or environment_id != 0):
+        for job in defined_jobs.values():
+            if project_id == job.project_id and environment_id == job.environment_id:
+                continue
+            else:
+                logger.error(
+                    f"❌ project_id, environment_id provided as arguments does not match the ID's in Jobs YAML file"
+                )
+                return None
```

Comment:
I understand the code but I am wondering if the logic is the one we would want.

My original idea was that we would _skip_ the jobs that don't match the definition and not raise an error. It feels more flexible for me. 

And if we do this, we should make sure thought that we are logging either with logger, or maybe when we show the change set in the terminal that some jobs are skipped because they didn't match.

Reply:
By adding the project_id and environment_id parameters as arguments to the commands, users now have two places where they can specify these ID's:
1. in the CLI while invoking plan, sync, import-jobs & 
2. in the YAML file job definition
In terms of design, I feel we should minimize the confusion for the user. If the user provides an ID in CLI, the expectation is that the jobs in that project/environment are being updated, and the user thinks of the ID's he passed as the operational scope. 
So, if there is a mismatch between the IDs provided in YAML and CLI, I want to flag it so the user can fix it and re-run the command.